transports/sftp/src/test/java/org/mule/transport/sftp/SftpReconnectionTestCase.java

```diff
@@ -6,25 +6,24 @@
  */
 package org.mule.transport.sftp;
 
-import org.mule.api.MuleException;
-import org.mule.context.notification.ConnectionNotification;
-import org.mule.tck.listener.ConnectionListener;
-import org.mule.tck.listener.FlowExecutionListener;
-
-import com.jcraft.jsch.SftpException;
-
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.Collection;
 
+import com.jcraft.jsch.SftpException;
 import org.junit.Test;
 import org.junit.runners.Parameterized;
+import org.mule.api.MuleException;
+import org.mule.context.notification.ConnectionNotification;
+import org.mule.tck.listener.ConnectionListener;
+import org.mule.tck.listener.FlowExecutionListener;
 
 public class SftpReconnectionTestCase extends AbstractSftpTestCase
 {
 
-    private static final long RECONNECTION_FREQUENCY = 1000l;
+    private static final long DELTA = 20l;
+    private static final long RECONNECTION_FREQUENCY = 1000l - DELTA;
```

Comment:
there is a hamcrest matcher for this, instead of modifying the value to verify (IsCloseTo)